<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtHill</class>
 <widget class="QDialog" name="qtHill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlainLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PlainLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plain (string)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CipherLineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CipherLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cipher (string)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key11Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key12Label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key21Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key22Label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key13Label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key33Label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key32Label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key31Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Key23Label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="KeyLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Key (matrix of integers)</string>
   </property>
  </widget>
  <widget class="QLabel" name="KeysizeLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose key size</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="OutputTextBrowser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OutputLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DecryptPushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EncryptPushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnalyzePushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qtHill</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qtHill</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
